src/main/resources/fileTemplates/j2ee/forge/Forge (1.13+) Submodule build.gradle.ft

```diff
@@ -85,12 +89,43 @@ minecraft {
             // Please read: https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels
             property 'forge.logging.console.level', 'debug'
 
+            #if (${GAME_TEST_FRAMEWORK})
+                property 'forge.enabledGameTestNamespaces', '${ARTIFACT_ID}'
+            #end
+
+            mods {
+                ${MOD_NAME} {
+                    source sourceSets.main
+                }
+            }
+        }
+#if (${GAME_TEST_FRAMEWORK})
+
+        gameTestServer {
+            workingDirectory project.file('run')
+
+            // Recommended logging data for a userdev environment
+            // The markers can be added/remove as needed separated by commas.
+            // "SCAN": For mods scan.
+            // "REGISTRIES": For firing of registry events.
+            // "REGISTRYDUMP": For getting the contents of all registries.
+            property 'forge.logging.markers', 'REGISTRIES'
+
+            // Recommended logging level for the console
+            // You can set various levels here.
+            // Please read: https://stackoverflow.com/questions/2031163/when-to-use-the-different-log-levels
+            property 'forge.logging.console.level', 'debug'
+
+            // Comma-separated list of namespaces to load gametests from. Empty = all namespaces.
+            property 'forge.enabledGameTestNamespaces', '${ARTIFACT_ID}'
```

Comment:
Why isn't this using MOD_NAME?

Reply:
Because it's necessary to use modid

Reply:
In your [gist](https://gist.github.com/SizableShrimp/60ad4109e3d0a23107a546b3bc0d9752), I have this. So I deduced that it was more ARTIFACT_ID to use than MOD_NAME
![image](https://user-images.githubusercontent.com/40738104/156815395-0e0e709b-1a40-4462-8e45-e9741b8689c2.png)